### PR TITLE
[Fedorovskyi] sequence_task_1

### PR DESCRIPTION
**Pull request by tra1ning5**

### CODE REVIEW

File: sequence_task_1/student/include/impl.h [hunk 1]
```diff
@@ -9,32 +9,82 @@ class StaticPlaylist
...
+    {
+        m_tracklist.emplace_back(std::forward<Args>(songData)...);
+        return m_tracklist.back();
+    }
 
     /** @todo Add track */
-    const Song_t& play(const Song_t& song);
+    const Song_t& play(const Song_t& song)
+    {
+        return *(m_tracklist.insert(m_tracklist.end(), song));
```
Comment (smay1613) [inline sequence_task_1/student/include/impl.h:62]: use push_back for insertion to the end

File: sequence_task_1/student/include/impl.h [hunk 2]
```diff
@@ -9,32 +9,82 @@ class StaticPlaylist
...
+        return *this;
+    }
 
     /** @todo Add track from initializer */
     template<class... Args>
-    const Song_t& play(Args&&... songData);
+    const Song_t& play(Args&&... songData)
+    {
+        m_tracklist.emplace_back(std::forward<Args>(songData)...);
+        return m_tracklist.back();
```
Comment (smay1613) [inline sequence_task_1/student/include/impl.h:56]: you can reuse current() method

File: sequence_task_1/student/include/impl.h [hunk 3]
```diff
@@ -9,32 +9,82 @@ class StaticPlaylist
...
+    const Song_t& current() const
+    {
+        return m_tracklist.back();
+    }
 
     /** @todo Skip to the next track in playlist, remove current */
-    void switchNext();
+    void switchNext()
+    {
+        m_tracklist.erase(std::prev(m_tracklist.end()));
```
Comment (smay1613) [inline sequence_task_1/student/include/impl.h:74]: you should use pop_back for last element erasure

File: sequence_task_1/student/include/impl.h (hunk 1 above)
Comment (tra1ning5) [reply sequence_task_1/student/include/impl.h]: Done

File: sequence_task_1/student/include/impl.h (hunk 2 above)
Comment (tra1ning5) [reply sequence_task_1/student/include/impl.h]: Done

File: sequence_task_1/student/include/impl.h (hunk 3 above)
Comment (tra1ning5) [reply sequence_task_1/student/include/impl.h]: Done

**Pull request closed without merging**